actions/access_keys/get.go

```diff
@@ -48,3 +48,43 @@ func (a *Action) get(c *gin.Context) {
 	contract := mappings.MapToAccessKeyContract(accessKey)
 	c.JSON(http.StatusOK, contract)
 }
+
+// get will get an existing model
+// Get access key godoc
+// @Summary		Get access key
+// @Description	Get access key
+// @Tags		Access-key
+// @Produce		json
+// @Param		id path string true "id of the access key"
+// @Success		200	{object} models.AccessKey "AccessKey with given id"
+// @Failure		400	"Bad request - Missing required field: id"
+// @Failure		403	"Forbidden - Access key is not owned by the user"
+// @Failure 	500	"Internal server error - Error while getting access key"
+// @Router		/v1/users/current/keys [get]
+// @Security	x-auth-xpub
```

Comment:
Same as for `Create access key`: use single method adjust description and use `// @DeprecatedRouter`

Reply:
done